nistats/datasets.py

```diff
@@ -283,35 +283,7 @@ def fetch_openneuro_dataset(
     return data_dir, sorted(downloaded)
 
 
-def _make_events_file_localizer_first_level(events_file):
-    """ Makes the first-level localizer fMRI dataset events file
-    BIDS compliant. Overwrites the original file.
-        Adds headers in first row.
-        Removes first column (spurious data).
-        Uses Tab character as value separator.
-    
-    Parameters
-    ----------
-    events_file: string
-        path to the localizer_first_level dataset's events file.
-    
-    Returns
-    -------
-    None
-    """
-    events = pd.read_csv(events_file, sep=' ', header=None, index_col=None,
-                           names=['session', 'trial_type', 'onset'],
-                           )
-    events.drop(labels='session', axis=1, inplace=True)
-    # duration is required in BIDS specification
-    events['duration'] = np.ones_like(events.onset)
-    # if events_file is open file handle, reset cursor to file beginning.
-    if hasattr(events_file, 'read') or hasattr(events_file, 'write'):
-        events_file.seek(0)
-    events.to_csv(events_file, sep='\t', index=False)
-
-
-def fetch_localizer_first_level(data_dir=None, verbose=1):
+def fetch_localizer_first_level(data_dir=None, url=None, verbose=1):
```

Comment:
Why have a `url` parameter? Is it our intention to allow users to specify their own URLs for fetching the data?
I believe we have had this discussion before and agreed o not include the url as a user facing option.